### PR TITLE
Moving CSV-/Raster-interface back to FileIO

### PR DESCRIPTION
**Pull request by rinkk**
As discussed, interfaces for external file formats are moved back into Applications/FileIO.

### CODE REVIEW

File: MeshGeoToolsLib/GeoMapper.cpp [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 #include <logog/include/logog.hpp>
 
-#include "GeoLib/IO/AsciiRasterInterface.h"
```
Comment (norihiro-w) [inline MeshGeoToolsLib/GeoMapper.cpp]: don't make a general lib dependent on FileIO

Comment (rinkk) [inline MeshGeoToolsLib/GeoMapper.cpp]: I'll fix the mapper class.

Comment (rinkk) [inline MeshGeoToolsLib/GeoMapper.cpp]: Removed the IO include.

**Comment by TomFischer** [comment]
The last commit can be merged into the first commit in the rebase process. :+1:

**Comment by norihiro-w** [comment]
:+1:

**Pull request merged**